remote-method/USAGE

```diff
@@ -0,0 +1,6 @@
+Description:
+  Creates a new remote method in the LoopBack application.
+
+Example:
+
+  slc loopback:remote-method
```

Comment:
> I don't want to explain inconsistent naming in docs/examples and the often overloaded "function/method" terminlogy in discussions. Unless you deem it is safe to assume that all end users will know method = remote-method. IMO, I'd rather keep remote-method = Remote method everywhere. No explaining differences etc.

I see, that makes sense. I guess I have lived too long inside the LoopBack bubble and cannot see this difficulties anymore.

Let's stick to `loopback:remote-method` then.